cpp/texture.cpp

```diff
@@ -36,6 +37,9 @@ GLint base_texture, mask_texture, base_coord, mask_coord, show_mask;
 Texture::Texture(int width, int height, void *data)
 	:
 	use_metafile{false} {
+		
+	assert(glGenBuffers != nullptr && "gl not initialized properly");
```

Comment:
unnecessary string? i would've thought "gl not initialized properly" would be obvious from "glGenBuffers != nullptr" ?